templates/ts-fastify-mongodb-backend/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "graphback-templates-mongodb-runtime",
+  "version": "1.0.1",
+  "description": "Graphback runtime template with Apollo Server and MongoDb",
```

Comment:
We are using Fastify in this template and not Apollo, let's change this to reflect that.